src/Skia/Avalonia.Skia/TypefaceCache.cs

```diff
@@ -62,7 +69,7 @@ private static SKTypeface GetTypeface(string name, FontKey key)
             {
                 typeface = SKTypeface.FromFamilyName(familyKey, key.Weight, SKFontStyleWidth.Normal, key.Slant);
 
-                if (typeface.FamilyName != name)
+                if (typeface?.FamilyName != name)
```

Comment:
I don't think this was initially implemented correctly, we shouldn't check for matching the font name, as Skia know better which font is more close to the font we are searching for, for me better this line be:
``` C#
if (typeface == null)
```

Reply:
Good point. I thought this would be unwanted behavior but it makes sense to change it that way because of the way FontConfig works. This change should make it possible to use an alias.